lib/tests/ErrorTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require_once('tests.inc');
+
+final class ErrorTest extends PHPUnit_Framework_TestCase {
+    private static $crt = null;
+    public static function setUpBeforeClass() {
+        self::$crt = new CRT();
+    }
+
+    public static function tearDownAfterClass() {
+        self::$crt = null;
+    }
+
+    public function testCanResolveErrorName() {
+        $this->assertEquals("AWS_ERROR_SUCCESS", CRT::error_name(0));
```

Comment:
"ERROR SUCCESS" ? :)

Reply:
error_name returns the enum name of the error. It's always going to be AWS_ERROR_XXX